CMakeLists.txt

```diff
@@ -26,13 +26,13 @@ endif()
 message(STATUS "[ATSDK] ATSDK_AS_SUBPROJECT: ${ATSDK_AS_SUBPROJECT}")
 
 # install each package
+message(STATUS "Building atlogger")
+add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/packages/atlogger)
+
```

Comment:
Had to move this to before atchops since atchops depends on atlogger now

Reply:
+1